### ISSUE TITLE
Deelgeed gone 

### ISSUE THREAD

**Issue by simthra** [HomepointXI/Issues#423]
**Date & Time**:  mar 24 2020 11 PM gmt+1

**Client Version (use `/ver` in game):** 30191204_1

**Character Name:** Thranduil

**Discord Name (if different from in game):**

**Nation:** bastok 10

**Job(level)/Sub Job(level):** mnk99/war49

**NPC or Monster or item Name:** Deelgeed

**Zone name:** Abyssea Grauberg

**Coordinates (use `!where`):** X: -128.666, Y: -89.223, Z: 43.91

**Item tool link (for items issues only):**

**Multi-boxing? (multiple clients on same connection):** yes

**Steps to reproduce:** Killed him last night approx 11 PM and he hasn't repopped yet. He's supposed to be a timed spawn.
TY

**Any additional information:**

**Comment by EpicTaru** [comment]
And here we have the wondrous Deelgeed observing it in it's rarely witnessed hibernating period where it burrows underground until the hibernating period has ended!

![deelgeed](https://user-images.githubusercontent.com/26195580/77544508-145ca180-6e77-11ea-9741-6de5812c7f3a.png)

It has been woken now and is none too pleased, caution is advised when approaching it before it's had a chance to imbibe its morning coffee!

![deelgeed feexed](https://user-images.githubusercontent.com/26195580/77544549-250d1780-6e77-11ea-961d-48699e072d61.png)

TLDR: Mob decided to wander underground and stay in its nice, quiet, off-map comfy hidey-hole.  Mob is now back at its normal spawn area.
